tools/dragen-transcriptome/4.2.4/dragen-transcriptome__4.2.4.cwl

```diff
@@ -239,6 +239,89 @@ inputs:
     inputBinding:
       prefix: "--rrna-filter-contig="
       separate: False
+  read_trimmers:
+    label: read trimming
+    type: string?
+    doc: |
+      To enable trimming filters in hard-trimming mode, set to a comma-separated list of the trimmer tools 
+      you would like to use. To disable trimming, set to none. During mapping, artifacts are removed from all reads.
+      Read trimming is disabled by default.
+    inputBinding:
+      prefix: "--read-trimmers"
```

Comment:
inputBinding should be of the format:

```
prefix: "--parameter-key="
separate: False
```

Reply:
Updated in https://github.com/umccr/cwl-ica/pull/391/commits/8c69f8334868ef36364852786fc37e3e1f8cdb22